omaha_server/omaha/request.xsd

```diff
@@ -90,7 +91,10 @@
             <xs:attribute name='experiments' use='optional' type="xs:string"/>
             <xs:attribute name='iid' use='optional' type="GuidType"/>
             <xs:attribute name='installage' use='optional' type='xs:integer'/>
+            <xs:attribute name='installdate' use='optional' type='xs:integer'/>
             <xs:attribute name='tag' use='optional' type='xs:string'/>
+            <xs:attribute name='_numaccounts' use='optional' type='xs:string'/>
+            <xs:attribute name='_numsignedin' use='optional' type='xs:string'/>
```

Comment:
It's good idea to add additional rules:

```
<xs:attribute name='cohort' use='optional' type='xs:string'/>
<xs:attribute name='cohorthint' use='optional' type='xs:string'/>
<xs:attribute name='cohortname' use='optional' type='xs:string'/>
```
They are may appeared in client request.